docs/source/guide/install.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Install and upgrade Label Studio
+type: guide
+order: 200
+---
+
+Install Label Studio on premises or in the cloud. Choose the install method that works best for your environment:
+- [Install with pip](#Install-with-pip)
+- [Install with Docker](#Install-with-docker)
+- [Install from source](#Install-from-source)
+- [Install with Anaconda](#Install-with-Anaconda)
+- [Install for local development](#Install-for-local-development)
+- [Upgrade Label Studio](#Upgrade-Label-Studio)
+
+
+## System requirements
+You can install Label Studio on a Linux, Windows, or MacOSX machine running Python 3.5 – 3.8. Python 3.9 is not yet supported. 
```

Comment:
actually Python 3.9 is working with anaconda

Reply:
I committed local changes for this and forgot to push them up 🤦🏻‍♀️